summary.tex

```diff
@@ -0,0 +1,16 @@
+\section{Summary}
+
+Rubin Observatory is putting in place a dedicated {\it Early Science Program} to ensure high-impact science during the first year of operation of the \lsst.
+This Early Science Program is motivated by the fact that in the current baseline there will be no science-ready data products released before \drone. 
+
+One of the four science pillars of LSST is time-domain astronomy, which is enabled by alerts on LSST detections of transient, variable, and/or moving objects. 
```

Comment:
This is not quite right: the science pillar is "exploring the transient & variable sky."  All three of the of the other science pillars involve time-domain astronomy (e.g., by identifying solar system objects, or SNe for cosmology).

Reply:
```suggestion
Time-domain astronomy is a key component of LSST's four science pillars and is enabled by alerts on LSST detections of transient, variable, and/or moving objects. 
```